Framework/LiveData/src/Kafka/KafkaEventStreamDecoder.cpp

```diff
@@ -133,7 +133,7 @@ KafkaEventStreamDecoder::KafkaEventStreamDecoder(
  * Destructor.
  * Stops capturing from the stream
  */
-KafkaEventStreamDecoder::~KafkaEventStreamDecoder() {}
+KafkaEventStreamDecoder::~KafkaEventStreamDecoder() { stopCapture(); }
```

Comment:
Not clear to my why moving the call to `stopCapture` here from the base class fixes anything.  Is that because it shouldn't be called for other types of `IKafkaStreamDecoder`?